src/components/CallView/shared/VideoVue.vue

```diff
@@ -361,6 +339,12 @@ export default {
 			}) || {}
 		},
 
+		participantActorType() {
+			return this.participant?.actorType || this.peerData.actorType || this.participantUserId
+				? ATTENDEE.ACTOR_TYPE.USERS
+				: ATTENDEE.ACTOR_TYPE.GUESTS
```

Comment:
?

`return this.$store.getters.getActorType()`

Reply:
Store getter is from actorStore and used only for defining self.
For VideoVue we need to get the type of other participant

Reply:
Should be fixed though for proper return:
```js
if (this.participant?.actorType) {
  return this.participant.actorType
} else if (this.peerData?.actorType) {
  return this.peerData.actorType
} else {
  return this.participantUserId
    ? ATTENDEE.ACTOR_TYPE.USERS
    : ATTENDEE.ACTOR_TYPE.GUESTS
}
```